patitasuy/src/app/components/edit-profile/edit-profile.component.ts

```diff
@@ -8,10 +11,14 @@ import { ModalController } from '@ionic/angular';
 })
 export class EditProfileComponent implements OnInit {
 
-  name: string = "prueba";
-  email: string = "prueba2";
+  name: string = "";
+  email: string = "";
```

Comment:
Is there still need for these fields to be here? Are you not using the form's fields?

Reply:
I forgot to delete it.